### DISCUSSION TITLE
heat conductionthrough different solid layers with HT3D

### DISCUSSION THREAD

**Discussion by cunconbkhp** [firemodels/fds#12928] [General]
Hello! I'd like to better understand the HT3D feature in FDS.

i want to simulate the heat transfers through multi solid layers. Each layer is made from a diffrent material.

Let take a simple case with two materials A and B. a bloc of A is neighbored to a bloc of B in Z direction (at Z=2.6). To use HT3D the code below will work : 

&SURF ID='A', HT3D=T/
&SURF ID='B', HT3D=T/

&OBST ID='Obstruction', XB=4.0,13.0,-0.5,0,2.6,2.75, HT3D=T, MATL_ID='CONCRETE', SURF_ID='A'/ 

&OBST ID='Obstruction', XB=4.0,13.0,-0.5,0,2.55,2.6, HT3D=T, MATL_ID='STEEL', SURF_ID='B'/ 

However in some fds example, the feature 'entre' is used to connect two solid layers, like below :
&SURF ID='A', HT3D=T/
&SURF ID='B', HT3D=T/
&SURF ID='entre'/

&OBST ID='Obstruction', XB=4.0,13.0,-0.5,0,2.6,2.75, HT3D=T, MATL_ID='CONCRETE', SURF_ID6='A','A','A','A','entre','A'/ 

&OBST ID='Obstruction', XB=4.0,13.0,-0.5,0,2.55,2.6, HT3D=T, MATL_ID='STEEL', SURF_ID6='B','B','B','B','B','entre'/ 

What are the diffrences of the two above approaches? 

Thank all in advance for your help!

**Comment by mcgratta** [comment]
The second case will not work. The obstructions are going to conform to the gas phase grid. Youi can put a lining material on the top of A or the bottom of B, but not in between.

**Comment by cunconbkhp** [comment]
Hi mcgratta. Thanks for your answer.


I atttached my test case. When i run this file, i could see the transfer of heat through the layers. So it works.

I dont really understand well the problem with "entre" surface.

Could you please take a look and give me some advices

[HT_3D_CHAUSSEE_x_HQD.fds.txt](https://github.com/firemodels/fds/files/15311783/HT_3D_CHAUSSEE_x_HQD.fds.txt)

**Comment by mcgratta** [comment]
I take it that "entre" is the French word for "between." When you do a 3D heat conduction calculation, you can only apply a `SURF_ID` to the exterior of a collection of obstructions. You cannot apply a `SURF` between obstructions.
